PWGCF/EbyEFluctuations/Tasks/kaonIsospinFluctuations.cxx

```diff
@@ -0,0 +1,2613 @@
+      recoK0s.fill(HIST("hK0s_3_10_K0s_Pi_Id1_Axis_00_21"), negDaughterTrack.p(), negDaughterTrack.tofNSigmaPi());             // Axis_p             ;
+      recoK0s.fill(HIST("hK0s_3_10_K0s_Pi_Id1_Axis_01_21"), negDaughterTrack.pt(), negDaughterTrack.tofNSigmaPi());            // Axis_pt            ;
+      recoK0s.fill(HIST("hK0s_3_10_K0s_Pi_Id1_Axis_02_21"), negDaughterTrack.tpcInnerParam(), negDaughterTrack.tofNSigmaPi()); // Axis_tpcInnerParam ;
+      recoK0s.fill(HIST("hK0s_3_10_K0s_Pi_Id1_Axis_03_21"), negDaughterTrack.tofExpMom(), negDaughterTrack.tofNSigmaPi());     // Axis_tofExpMom     ;
+      recoK0s.fill(HIST("hK0s_3_10_K0s_Pi_Id1_Axis_20_21"), negDaughterTrack.tpcNSigmaPi(), negDaughterTrack.tofNSigmaPi());   // Axis_tpcInnerParam ;
+    }
+    //
+
+    int v0daughterCollisionIndexTag = 0;
+    if (v0.collisionId() == posDaughterTrack.collisionId()) {
+      v0daughterCollisionIndexTag = +1;
+    }
+    if (v0.collisionId() == negDaughterTrack.collisionId()) {
+      v0daughterCollisionIndexTag = +2;
+    }
+    if (posDaughterTrack.collisionId() == negDaughterTrack.collisionId()) {
+      v0daughterCollisionIndexTag = +4;
+    }
+
+    recoK0s.fill(HIST("hK0s_4_01_v0DaughterCollisionIndexTag"), v0daughterCollisionIndexTag);
+  }
+
+  template <typename T>
+  void FindRepeatEntries(std::vector<int64_t> ParticleList, T hist)
+  {
+    for (uint ii = 0; ii < ParticleList.size(); ii++) {
+      int nCommonCount = 0; // checking the repeat number of track
+      for (uint jj = 0; jj < ParticleList.size(); jj++) {
+        if (ParticleList[jj] == ParticleList[ii]) {
+          if (jj < ii) {
+            break;
+          } // break if it was already counted
+          nCommonCount++; // To Calculate no of times the entry was repeated
+        }
+      }
+      hist->Fill(nCommonCount);
+    }
+  }
+
+  void FillNewListFromOldList(std::vector<int64_t>& NewList, std::vector<int64_t> OldList)
+  {
+    for (uint ii = 0; ii < OldList.size(); ii++) {
+      bool RepeatEntry = false;
+      for (uint jj = 0; jj < NewList.size(); jj++) {
+        if (OldList[ii] == NewList[jj]) {
+          RepeatEntry = true;
+        }
+      }
+      if (!RepeatEntry) {
+        NewList.push_back(OldList[ii]);
+      }
+    }
+  }
+
+  template <typename T>
+  bool checkTrackSelection(T track, std::vector<int64_t> DauParticleList, uint& skippingPosition, int& rejectionTag)
+  {
+    if (track.tpcNClsCrossedRows() < 70) {
+      rejectionTag = 1;
+      return false;
+    }
+    if (fabs(track.dcaXY()) > 0.2) {
+      rejectionTag = 2;
+      return false;
+    }
+    if (!track.isGlobalTrack()) {
+      rejectionTag = 3;
+      return false;
+    }
+
+    bool FlagDaughterTrack = false;
+    if (track.globalIndex() == DauParticleList[skippingPosition]) {
+      FlagDaughterTrack = true;
+      skippingPosition++;
+      // LOG(info) <<"*******************iEvent = "<<iEvent<<" :: ID = "<<collIdx<<" :: index skipping = "<<track.globalIndex();
```

Comment:
Don't leave commented code
In this case the code might be left uncommented if `debug` is used instead of `info`